source/regions/flags.rst

```diff
@@ -37,13 +37,21 @@ The output of this command is interactive in-game. Click flag values to change t
 Region Groups
 =============
 
-Sometimes, it may be desired for a flag to only apply to a certain group of players rather than everyone that should enter the region. This can be achieved by specifying an additional "region group" when defining the flag, of which there are several options:
+Sometimes, it may be desired for a flag to only apply to a certain group of players rather than everyone that should enter the region. This can be achieved by specifying an additional "region group" when defining the flag.
 
-* all (everyone)
-* members
-* owners
-* nonmembers
-* nonowners
```

Comment:
I turned this into a table with short descriptions, since I've noticed some people struggling with what region groups are supposed to be